src/kernel/lion_step/cuda.cu

```diff
@@ -0,0 +1,115 @@
+/*! @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+ *                              (Skoltech), Russia. All rights reserved.
+ *                 2023-present Artificial Intelligence Research Institute
+ *                              (AIRI), Russia. All rights reserved.
+ *
+ * NNTile is software framework for fast training of big neural networks on
+ * distributed-memory heterogeneous systems based on StarPU runtime system.
+ *
+ * @file src/kernel/adam_step/cuda.cu
```

Comment:
this entire file is just an unchanged adam step, why would one put it onto pull request?